skyportal/handlers/api/public_pages/public_source_page.py

```diff
@@ -21,6 +21,10 @@ def calculate_hash(data):
 
 
 def process_thumbnails(thumbnails, ra, dec):
+    thumbnails = sorted(
+        thumbnails,
+        key=lambda x: ["new", "ref", "sub", "sdss", "ls", "ps1"].index(x["type"]),
```

Comment:
This should source the enum found here, not redefine one: https://github.com/skyportal/skyportal/blob/main/skyportal/enum_types.py#L52